.circleci/config.yml

```diff
@@ -2,7 +2,7 @@ version: 2.1
 
 docker_defaults: &docker_defaults
   docker:
-    - image: circleci/node:8.10.0
+    - image: circleci/node:11.15.0
```

Comment:
Any reason for changing the version? We run version 8 in production: https://github.com/all-contributors/all-contributors-bot/blob/master/serverless.yml#L33

Reply:
yes, for `chalk 4` requiring `node >=10` it appears

Reply:
@jakebolam https://github.com/all-contributors/all-contributors-bot/pull/305#pullrequestreview-400516419